client_end_user/shared/src/commonMain/kotlin/presentation/composable/ImageSlider.kt

```diff
@@ -0,0 +1,119 @@
+package presentation.composable
+
+
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.animateOffsetAsState
+import androidx.compose.animation.core.spring
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.PagerState
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.layout.positionInParent
+import androidx.compose.ui.unit.dp
+import com.beepbeep.designSystem.ui.theme.Theme
+import presentation.composable.modifier.noRippleEffect
+
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun ImageSlider(
+    images: List<String>,
+    modifier: Modifier = Modifier,
+    indicatorColor: Color = Theme.colors.primary,
+    indicatorAlignment: Alignment = Alignment.BottomEnd,
+    onItemClickListener: (id: Int) -> Unit
```

Comment:
```suggestion
    images: List<String>,
    onItemClickListener: (id: Int) -> Unit,
    modifier: Modifier = Modifier,
    indicatorColor: Color = Theme.colors.primary,
    indicatorAlignment: Alignment = Alignment.BottomEnd,
```

For match compose structure